src/globals/scss/_css--body.scss

```diff
@@ -4,72 +4,8 @@
 @import 'typography';
 @import 'import-once';
 
-@mixin typography {
-  h1 {
-    @include reset;
-    @include font-size('32');
-    @include line-height('heading');
-    @include letter-spacing;
-  }
-
-  h2 {
-    @include reset;
-    @include font-size('26');
-    @include line-height('heading');
-    @include letter-spacing;
-  }
-
-  h3 {
-    @include reset;
-    @include font-size('20');
-    @include line-height('heading');
-    @include letter-spacing;
-  }
-
-  h4 {
-    @include reset;
-    @include font-size('18');
-    @include line-height('heading');
-  }
-
-  h5 {
-    @include reset;
-    @include font-size('16');
-    @include line-height('heading');
-  }
-
-  h6 {
-    @include reset;
-    @include font-size('14');
-    @include line-height('heading');
-  }
-
-  p {
-    @include reset;
-    @include font-size('16');
-    @include line-height('body');
-  }
-
-  strong {
-    @include reset;
-    @include font-smoothing;
-    @include letter-spacing;
-    font-weight: 700;
-  }
-
-  em {
-    @include reset;
-    font-style: italic;
-  }
-
-  a {
-    @include reset;
-    color: $brand-01;
-  }
-}
-
 @mixin css-body {
-  .bx--body {
+  body {
```

Comment:
Settings `$css--body: true` will output these `body` styles to CSS. 
This is the default.